packages/rum-core/src/performance-monitoring/capture-hard-navigation.js

```diff
@@ -208,11 +233,18 @@ function captureHardNavigation(transaction) {
         }
         ajaxUrls.push(span.name.split(' ')[1])
       }
-      createResourceTimingSpans(entries, ajaxUrls).forEach(function(span) {
+      createResourceTimingSpans(resourceEntries, ajaxUrls).forEach(function(
+        span
+      ) {
         if (isValidSpan(transaction, span)) {
           transaction.spans.push(span)
         }
       })
+
+      const userEntries = perf.getEntriesByType('measure')
+      createUserTimingSpans(userEntries).forEach(
+        span => span._end <= transaction._end && transaction.spans.push(span)
```

Comment:
Let's apply the same validation done in #277!

Reply:
I will rebase and add once the other PR is merged.